aws_s3_bucket.tf

```diff
@@ -10,6 +10,10 @@ resource "aws_s3_bucket" "noS3BucketSseRules" {
     Name        = "nos3BucketSseRules"
     Environment = "Dev"
   }
+
+  versioning {
```

Comment:
**Accurics remediation**
 
**[HIGH]: S3 bucket should have versioning enabled**
 
**Vulnerability**
Enabling S3 versioning will enable easy recovery from both unintended user actions, like deletes and overwrites
 
**Remediation**
S3 bucket should have versioning enabled

source: accurics-